src/Covid19Dashboard/PlotChart/index.jsx

```diff
@@ -40,20 +60,48 @@ function formatChartData(plots) {
   });
   let sortedData = Object.values(dateToData);
   sortedData = sortedData.sort((a, b) => new Date(a.date) - new Date(b.date));
-  // console.log(sortedData);
-  return { data: sortedData }; // , max };
+  return { data: sortedData, ticks: getDates(sortedData[0].date, sortedData[sortedData.length - 1].date, 7) }; // , max };
 }
 
+// 10 colors generated with
+// https://medialab.github.io/iwanthue/
+// Intense + colorblind option + hard (force vector)
+const COLORS = ["#46151f", "#0078b8", "#00beb5", "#f5bc6e", "#006ad0", "#ff50a2", "#009b2b", "#91c600", "#6266ff", "#ff3cca",];
+
 class PlotChart extends PureComponent { // eslint-disable-line react/no-multi-comp
+  state = {
+    width: Object.fromEntries(
+      Object.entries(this.props.plots).map(([k, v], i) => [v.name, 1]),
+    ),
+  };
+
+  handleMouseEnter = (o) => {
+    const { dataKey } = o;
+    const { width } = this.state;
+
+    this.setState({
+      width: { ...width, [dataKey]: 2 },
+    });
+  }
+
+  handleMouseLeave = (o) => {
+    const { dataKey } = o;
+    const { width } = this.state;
+
+    this.setState({
+      width: { ...width, [dataKey]: 1 },
+    });
+  }
```

Comment:
could you add comments to explain what these 2 functions are doing

and maybe rename `width` to something like `nameToWidth` or whatever is relevant